torch/testing/_internal/common_methods_invocations.py

```diff
@@ -5744,6 +5744,30 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
            assert_autodiffed=True,
            supports_forward_ad=True,
            autodiff_nonfusible_nodes=['aten::mul'],),
+    OpInfo('__rand__',
```

Comment:
These are really cool. Would you also add some tests in test_binary_ufuncs.py that compares these operators with their tensor x scalar results? That is, a test that ensures that scalar x tensor and tensor x scalar is equivalent for all these operations?